compliance/admin.py

```diff
@@ -17,11 +18,22 @@ class ExportsInquiryLogAdmin(admin.ModelAdmin):
 
     model = ExportsInquiryLog
     search_fields = ["user__email", "computed_result", "info_code", "reason_code"]
-    list_filter = ["computed_result", "info_code", "reason_code"]
-    list_display = ["user", "computed_result", "info_code", "reason_code"]
+    list_filter = [
+        "computed_result",
+        "info_code",
+        "reason_code",
+        "user__legal_address__country",
+    ]
+    list_display = ["user", "computed_result", "info_code", "reason_code", "country"]
+    list_select_related = ["user__legal_address"]
     readonly_fields = [] if settings.DEBUG else get_field_names(ExportsInquiryLog)
     actions = ["manually_approve_inquiry"]
 
+    def country(self, instance):
+        """Get country name from ISO Alpha-2 country code"""
+        country = pycountry.countries.get(alpha_2=instance.user.legal_address.country)
+        return country.name if country else "N/A"
```

Comment:
It's ok go with that, but we don't have any such case where there will be no country alpha_2 as its already being validating while creating account.